### DISCUSSION TITLE
R version information

### DISCUSSION THREAD

**Discussion by RachBioHaz** [FrederickHuangLin/ANCOMBC#211] [General]
Hi, not sure if this information is already listed somewhere and I just missed it.  Is it possible to have listed what R version the most current ANCOMBC package is compatible with? Maybe just above the installation information in the README file?

**Comment by FrederickHuangLin** [comment]
Hi @RachBioHaz,

The latest `ANCOMBC` requires R (>= 4.3.0). You can find more information here: https://www.bioconductor.org/packages/release/bioc/html/ANCOMBC.html

Best,
Huang
